### PR TITLE
Added the User Information Form 

### LINKED ISSUE #1: Ability to add users information to the top of the resume
https://github.com/MLH-Fellowship/orientation-project-python/issues/22

### PR DESCRIPTION
**Pull request by esohel30**
closes #1 

**Added a new file UserInfo.js to manage the user information form.**

The form includes input fields for:
1. Name
2. Phone Number (with country code enforcement using a regex pattern)
3. Email Address
4. State management for the form inputs using React's useState hook.
5. Form validation for required fields and pattern enforcement for the phone number.

**Update to App.js:**

1, Imported the UserInfo component.
2. Integrated the UserInfo component at the top of the existing sections for Experience, Education, and Skills.
3. Ensured the overall layout remains consistent and the user experience is enhanced with a clear section for user information.

**Screenshots**

Screen Shot 2024-05-27 at 10 33 46 PM

### CODE REVIEW

**Comment by esohel30** [comment]
@Pradyuman7 tagging you here as I cannot assign you as a reviewer.

File: src/UserInfo.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
+.userInfoSection {
+    margin-bottom: 20px;
```
Comment (Pradyuman7) [inline src/UserInfo.css]: nit: it's generally better to use rem instead of px

Comment (esohel30) [reply src/UserInfo.css]: @Pradyuman7 Got it fixed.

**Review by Pradyuman7 — APPROVED** [review]
LGTM 🔥

**Pull request merged**